data/resources/ui/content-message-document.ui

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <template class="ContentMessageDocument" parent="ContentMessageBase">
+    <property name="layout-manager">
+      <object class="GtkBoxLayout">
+        <property name="orientation">vertical</property>
+      </object>
+    </property>
+    <style>
+      <class name="message-bubble" />
```

Comment:
```suggestion
      <class name="message-bubble"/>
```

Are you using vscode? If yes:
![image](https://user-images.githubusercontent.com/3630213/189496650-f3ad9651-02f6-4400-acc7-a3b1b895442f.png)

There are more in this file.